kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -1888,7 +1888,6 @@ private boolean isListOfVarsAttribute(String name) {
     private static String[] asciiReadableEncodingKoreCalc() {
         String[] koreEncoder = Arrays.copyOf(StringUtil.asciiReadableEncodingDefault, StringUtil.asciiReadableEncodingDefault.length);
         koreEncoder[0x26] = "And-";
-        koreEncoder[0x2d] = "-";
```

Comment:
I'm not sure removing this line is correct.

Reply:
From my understanding, this code is just adapting the `asciiReadableEncodingDefault`, and as we are already setting it as `-` [there](https://github.com/runtimeverification/k/blob/e91f273282c632cbd3b4f735ee8139251aa6ca25/kore/src/main/java/org/kframework/utils/StringUtil.java#L551), we don't have to change it in this function.